bosh-setup/azuredeploy.json

```diff
@@ -457,7 +515,7 @@
           "osDisk": {
             "name": "osdisk",
             "vhd": {
-              "uri": "[concat('http://',variables('defaultStorageAccountName'),'.blob.',variables('environment').serviceHostBase,'/',variables('vmStorageAccountContainerName'),'/',parameters('vmName'),'-osdisk.vhd')]"
+              "uri": "[concat('http://', variables('defaultStorageAccountName'), '.blob.', variables('environment').serviceHostBase, '/', variables('vmStorageAccountContainerName'), '/', parameters('vmName'), '-osdisk.vhd')]"
```

Comment:
Thanks @bmoore-msft .
I didn't know the best practice for the vhd uri. I think, we can follow the best practice.